src/inflators/spawner.ts

```diff
@@ -32,13 +33,15 @@ export function inflateSpawner(world: HubsWorld, eid: number, props: SpawnerPara
   addComponent(world, CursorRaycastable, eid);
   addComponent(world, RemoteHoverTarget, eid);
   addComponent(world, SingleActionButton, eid);
+  addComponent(world, TogglesHoveredActionSet, eid);
 
   addComponent(world, ObjectSpawner, eid);
   ObjectSpawner.src[eid] = APP.getSid(props.src);
   ObjectSpawner.flags[eid] = props.mediaOptions?.applyGravity ? OBJECT_SPAWNER_FLAGS.APPLY_GRAVITY : 0;
 
   inflateRigidBody(world, eid, {
     mass: 0,
+    type: Type.STATIC,
```

Comment:
Question for my study: Spawner object should be static (no local matrix update) so rigid body type should be static. Is my understanding correct?

Reply:
That's my understanding. Spawners are static as their main action is to spawn an object and are not draggable. Also that's how we were setting them up in AFrame:
https://github.com/mozilla/hubs/blob/5f6e7a62ab998a60895a30dc686ae6db94fe9c5b/src/gltf-component-mappings.js#L323